client/extensions/woocommerce/woocommerce-services/lib/initialize-carrier-accounts-state.js

```diff
@@ -0,0 +1,41 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+export default () => {
+	const requiredFields = [
+		'account_number',
+		'name',
+		'address',
+		'city',
+		'state',
+		'state',
```

Comment:
There are 2 states here. We should probably remove 1.

Reply:
Yep, already did that